### ISSUE TITLE
Recommend WCAG 2.1 or WCAG 2.0?

### ISSUE THREAD

**Issue by JamieKnight** [mmu-webdesign/mmu-webdesign.github.io#1]
Not sure which version of WCAG would be most appropriate, but I’d personally tend towards WCAG 2.1 because it’s latest version and I am already seeing projects require it.

This issue is more to start a discussion than provide an answer.

**Comment by eskins** [comment]
Agreed. We get the students to use the WebAim checklist (easier to understand than the actual spec). I see they've added 2.1 criteria. Another update for us! [https://webaim.org/standards/wcag/checklist](https://webaim.org/standards/wcag/checklist )
